### ISSUE TITLE
multifile upload field shouldn't include form-control class

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/curation_concerns#751]
By default hydra-editor wraps all inputs with form-control.
```
# hydra-editor-1.2.0/config/initializers/simple_form.rb
    3: SimpleForm.setup do |config|
    4:   config.wrappers :inline, tag: 'span', class: 'form-group inline', error_class: 'has-error' do |b|
    5:     b.use :html5
    6:     b.use :placeholder
    7:     b.use :label, class: 'control-label'
=>  8:     b.use :input, class: 'form-control'
```

This makes the upload field look like this:
<img width="735" alt="screen shot 2016-04-28 at 9 29 38 am" src="https://cloud.githubusercontent.com/assets/92044/14889482/9a2580ec-0d24-11e6-98b4-9796b6f472b4.png">

But we want it to look like this (`form-control` class removed)
<img width="366" alt="screen shot 2016-04-28 at 9 29 53 am" src="https://cloud.githubusercontent.com/assets/92044/14889504/a6626cda-0d24-11e6-870d-7a8fdcaa6b62.png">

**Comment by jcoyne** [comment]
This ought to fix it: 
https://github.com/projecthydra/hydra-editor/pull/111

**Comment by jcoyne** [comment]
Fixed by #756
